Source/EasyNetQ/Producer/IResponderConfiguration.cs

```diff
@@ -10,12 +11,20 @@ public class ResponderConfiguration : IResponderConfiguration
     {
         public ResponderConfiguration(ushort defaultPrefetchCount)
         {
+            Priority = 0;
```

Comment:
It is not necessary to assign 0.

Reply:
This is done the same way in ConsumerConfiguration class, so I was looking into ConsumerConfiguration as an example. I can remove this assignment.
https://github.com/EasyNetQ/EasyNetQ/blob/b7936a70a1b08e1666a208faaf1378d06fc41a09/Source/EasyNetQ/Consumer/IConsumerConfiguration.cs

Reply:
Well, they just do not make sense. All class fields are initialized with zeros.